lambdas/search/index.py

```diff
@@ -13,7 +13,8 @@
 from t4_lambda_shared.decorator import api
 from t4_lambda_shared.utils import get_default_origins, make_json_response
 
-MAX_QUERY_DURATION = '15s'
+DEFAULT_SIZE = 1_000
+MAX_QUERY_DURATION = '30s'
```

Comment:
Since API gateway's timeout is 30s, we should make sure Elastic returns with a few seconds to spare. Maybe try 25s instead?